core/src/smartcontracts/isi/block.rs

```diff
@@ -41,7 +44,11 @@ impl ValidQuery for FindBlockHeaderByHash {
         let block = wsv
             .all_blocks_by_value()
             .find(|block| block.hash() == hash)
-            .ok_or_else(|| QueryExecutionFailure::Find(Box::new(FindError::Block(hash))))?;
+            .ok_or_else(|| {
+                QueryExecutionFailure::Find(Box::new(FindError::Block(FinishedBlockHash {
+                    internal: *hash,
+                })))
+            })?;
```

Comment:
Looks like we searching by just `hash()` but error says about `FinishedBlockHash`

Reply:
This is reverted to the original now.